### PR TITLE
feat: init editor

### PR DESCRIPTION
**Pull request by hunmix**
init editor and mark class

### CODE REVIEW

File: src/typings/index.ts [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
-  type: "string" | "number" | "color" | "boolean";
-  default: any;
+export interface IVisualBaseConfig {
+  type: string | number | boolean;
+  default: string | number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   options?: any;
 }
+
+declare global {
```
Comment (purpose233) [inline src/typings/index.ts]: window.view is only used for debugging, maybe use eslint disble instead

File: src/view/editor/index.tsx [hunk 2]
```diff
@@ -15,26 +17,49 @@ export const Editor = () => {
...
+
+  useEffect(() => {
+    if (containerRef.current && !editorRef.current) {
+      editorRef.current = new MarksEditor({
+        container: containerRef.current,
+      })
+      editorRef.current.init()
+      window.editor = editorRef.current
+    }
+    // return () => editorRef.current?.release()
```
Comment (purpose233) [inline src/view/editor/index.tsx]: apply release here

File: src/typings/index.ts (hunk 1 above)
Comment (hunmix) [reply src/typings/index.ts]: fixed

File: src/view/editor/index.tsx (hunk 2 above)
Comment (hunmix) [reply src/view/editor/index.tsx]: fixed

**Review by purpose233 — APPROVED** [review]

**Pull request merged**